common/static/js/capa/src/formula_equation_preview.js

```diff
@@ -3,6 +3,37 @@ var formulaEquationPreview = {
     errorDelay: 1500  // Wait time before showing error (prevent frustration).
 };
 
+function escapeCSS(id) {
```

Comment:
I'm not sure what we want to do now, and what we want to ticket.
1. Doesn't seem right to have an escaping function without some tests. (recommend now)
2. This probably belongs in this (deprecated library).  Maybe that would be a follow-up?
https://github.com/edx/edx-ui-toolkit/blob/master/src/js/utils/html-utils.js  If not now, is there a better shared location in edx-platform if this sits for a while?
3. We should update the docs for CSS escaping: https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/preventing_xss/preventing_xss.html#css-context-and-escaping
4. Are there linting recommendations that we should add based on this find?

Reply:
1. 100% agreed. I'll look into it. For reference, since this is basically copying jQuery's escapeSelector, we can take inspiration from [upstream's tests](https://github.com/jquery/jquery/blob/3edfa1bcdc50bca41ac58b2642b12f3feee03a3b/test/unit/selector.js#L2030).
2. if that could be moved to a better location as a follow up, that would be good. At the moment it's only used in this one place, so it's kind of ok?
3. this is different to sanitizing user supplied css; CSS.escape is soley for escaping a string so it is valid in a css selector (eg. `button.login-button`). These was (afaik) no possibility of xss with how this was used before escaping the id in this PR.
4. That would require some thinking - perhaps this is something that one just needs to be aware of when they see untrusted/uncontrolled strings being used in css/jquery selectors.

Reply:
Tests added.